src/Map.elm

```diff
@@ -292,10 +339,13 @@ landscapeToString height =
     let
         (LandscapeId landscapeId) =
             heightToLandscapeId height
+
+        (Landscapes l) =
+            landscapes
     in
-    case Array.get landscapeId landscapes of
-        Just l ->
-            Tuple.first l
+    case Array.get landscapeId l of
+        Just ( LandscapeName name, res ) ->
+            name
 
         Nothing ->
             Debug.todo "Not landscape found :/"
```

Comment:
`Debug.todo` won't be available if you publish this as a package or if your want to build this with `--optimize`  This function should probably return a `Maybe` to indicate the possibility the array being empty. 
The other option is to pass in a https://package.elm-lang.org/packages/mgold/elm-nonempty-list/latest/ instead of an `Array` if you know it can't be empty.